lib/client/cluster_client.go

```diff
@@ -151,7 +165,7 @@ func (c *ClusterClient) SessionSSHConfig(ctx context.Context, user string, targe
 	log.Debug("Issued single-use user certificate after an MFA check.")
 	am, err := key.AsAuthMethod()
 	if err != nil {
-		return nil, trace.Wrap(err)
+		return nil, trace.Wrap(ceremonyFailedErr{err})
```

Comment:
why wrap it here?

And what underlying error type are we hiding that causes the issue?